Mlem/Extensions/Swipey Actions.swift

```diff
@@ -7,84 +7,63 @@
 
 import SwiftUI
 
+struct SwipeAction {
+    
+    struct Symbol {
+        let emptyName: String
+        let fillName: String
+    }
+    
+    let symbol: Symbol
+    let colour: Color
```

Comment:
Nit but we should probably go American with the spelling because that's how the library spells it

Reply:
ah yeah, muscle memory - I'll sort it 👍

Reply:
also, _American_ is a funny way to spell _wrong_ 😉

Reply:
sorted, added an `unfix` commit 😉